### ISSUE TITLE
Improving accessibility by fixing ARIA (Accessible Rich Internet Applications) compliance

### ISSUE THREAD

**Issue by jnaito777** [hackforla/food-oasis#1397]
### Overview
We need to use a chrome extension and identify improvements related to ARIA compliance so that we can create issues accordingly and improve accessibility of our application.

#### Details
Fixing ARIA on different parts of the web page to improve accessibility. For food seekers with disabilities, they rely on assistive technologies(like screen readers) to navigate the site(ex: having text labels for an object, like a button).

### Action Items
- [ ] Install Chrome Extension: https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh
- [ ] Document everything it finds wrong with screenshots and text details
- [ ] Review with product team
- [ ] Create Issues to fix each problem

### Resources/Instructions
- This issue was originally created from #748.

Original benchmark was done on Google Lighthouse. Accessibility snapshots below:
<details><summary>Homepage</summary>

<img width="832" alt="Screen Shot 2022-10-13 at 4 29 05 PM" src="https://user-images.githubusercontent.com/114040198/197943964-9910c537-b66c-4908-ad5a-5da9a790e93e.png">
</details> 

<details><summary>Map Page</summary>

<img width="802" alt="Screen Shot 2022-10-13 at 4 26 53 PM" src="https://user-images.githubusercontent.com/114040198/197943982-99dd8b65-5db2-4aa0-8ed3-d129a7bd41b0.png">
</details>

**Comment by staceyrebekahscott** [comment]
Moving to Prioritized Backlog, to be assessed once current Dev workload is completed.

**Comment by ExperimentsInHonesty** [comment]
We have created a new repo to help teams work on accessability issue https://github.com/hackforla/accessibility.  Its not 100% ready because it does not have all the issue templates.  In about a month, it should have all the templates this team needs to create the over arching Initiative issues, Epic issues, and sub issues to audit and fix all accessability problems.
